AppCenter/AppCenter/Internals/DelegateForwarder/MSCustomApplicationDelegate.h

```diff
@@ -9,7 +10,7 @@ NS_ASSUME_NONNULL_BEGIN
  * registered through swizzling. Due to the early registration of swizzling on the original app delegate each custom delegate must sign up
  * for selectors to swizzle within the `load` method of a category over the @see MSAppDelegateForwarder class.
  */
-@protocol MSCustomApplicationDelegate <NSObject>
+@protocol MSCustomApplicationDelegate <MSCustomDelegate>
```

Comment:
Yes, `MSCustomDelegate` is the base protocol now since we added an inheritance layer to the AppDelegate it makes sens to do the same on the protocol too. When we'll introduce `MSCustomUserNotificationCenterDelegate` it will also inherit from `MSCustomDelegate`.